ffi/dotnet-idsdk/IoticsIdLib/src/TwinIdentity.cs

```diff
@@ -0,0 +1,18 @@
+namespace IOTICS;
+
+public class TwinIdentity(Identity factory, string seed, string keyName, string id, string did) : PartyIdentity(factory, seed, keyName, id, did)
+{
+        public TwinIdentity(Identity factory, AgentIdentity creator, string keyName, string id, string did): this(factory, creator.Seed, keyName, id, did) {
+
+        }
```

Comment:
No re-create method for this one?

Reply:
It has not been exposed via ffi (TBD soon)

Reply:
.. but there is `RecreateTwinIdentity` in the FFI wrapper?